static/scripts/demo/auth-context.ts

```
@@ -368,6 +370,18 @@ Comment \`/demo\` below to initiate an interactive demonstration. Your AI team m
   return issue;
 }
 
+async function inviteUserAsCollaborator(octokit: Octokit, repo: { owner: { login: string }; name: string }) {
+  const botUserName = "ubiquity-os-simulant";
```

Can you throw an error in case we rename the bot so that its easy to debug?

---

If the bot name is renamed, octokit should throw an exception saying that the user could not be found